internal/application/filesystem.go

```diff
@@ -550,11 +559,23 @@ func runTranslate(appName string, artifactpath string, artifactName string, tran
 			translatech <- port
 			close(translatech)
 		}
-		log.Info(t)
+		if Verbose {
+			generateVerboseLogs(t)
+		} else {
+			log.Info(t)
+		}
 	}
 	return true
 }
 
+// generateVerboseLogs synchronizes move2kube-api loggging level wrt move2kube logging level
+func generateVerboseLogs(message string) {
+	var re = regexp.MustCompile(loglevelRegex)
+	sm := re.FindStringSubmatch(message)
+	loggingLevel := sm[1]
```

Comment:
Nice work! Do you want to check here the number of segments, just to prevent unnecessary panics.

Reply:
Yes sure.